src/main/java/org/littil/api/location/repository/AbstractLocationEntity.java

```diff
@@ -0,0 +1,24 @@
+package org.littil.api.location.repository;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import org.littil.api.auditing.repository.AbstractAuditableEntity;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+
+@Getter
+@Setter
+@ToString(callSuper = true)
+@MappedSuperclass
+@EntityListeners(LocationEntityListener.class)
+public abstract class AbstractLocationEntity extends AbstractAuditableEntity {
```

Comment:
Why did you choose for an abstract class with these properties?

Reply:
For no good reason ;)